sdk/openai/openai-rest/README.md

```diff
@@ -6,7 +6,7 @@ Azure OpenAI APIs for completions and search
 
 Key links:
 
-- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/openai/rlc-generated/sdk/openai/openai-rest)
+- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/openai/openai-rest)
 - [Package (NPM)](https://aka.ms/npm-azure-rest-openai)
 - [API reference documentation](https://aka.ms/AAp1kxa)
 - [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/openai/openai/samples)
```

Comment:
Samples still points to `openai/openai/samples`. Are we going to have new samples for `openai-rest`?

Reply:
I believe Deyaa is working on them. cc/ @deyaaeldeen

Reply:
Thanks for the fix! All that was being addressed in https://github.com/Azure/azure-sdk-for-js/pull/28501 but still wanted to have a few discussions before merging it.